docs/scos/dev/back-end-development/data-manipulation/data-publishing/handling-data-with-publish-and-synchronization.md

```diff
@@ -662,7 +662,7 @@ class HelloWorldStorageDeleter implements HelloWorldStorageDeleterInterface
      *
      * @return void
      */
-    protected function delete(int $idMessage): void
+    private function delete(int $idMessage): void
```

Comment:
This should be reverted. We do not promote unnecessary visibility.

Reply:
I'll do that, but can you explain why? For project code, `final` classes and `private` methods for internal stuff are the defaults for me.